### DISCUSSION TITLE
如何监听到连接桩连接到画布空白处？

### DISCUSSION THREAD

**Discussion by G-Veigar** [antvis/X6#2251] [Q&A] [answered]
需求背景：从连接桩拖出连接线到画布空白处时，给一个快速创建节点的弹框。类似processon的功能，如下。
<img width="242" alt="image" src="https://user-images.githubusercontent.com/19259903/174797842-41c633f6-8722-4442-95ab-2dda8b2a91e3.png">

问题是，开启了connecting.allowBlank，怎么才能监听到触发了连接到空白处的事件呢？

**Comment by NewByVector** [comment, marked as answer]
可以在 validateEdge 里面根据 edge 的 target 来判断是否连接到空白处。
